Pinta.Core/ImageFormats/IImageImporter.cs

```diff
@@ -36,5 +36,5 @@ public interface IImageImporter
 	/// <param name='parent'>
 	/// Window to be used as a parent for any dialogs that are shown.
 	/// </param>
-	void Import (Gio.File file, Gtk.Window parent);
```

Comment:
The doc comment `Imports a document into Pinta` could be updated to something like `Loads a new document from a file` since all it's doing now is creating a document and other code handles registering it with Pinta

Reply:
This is a good point. I changed the comment and also removed the `Gtk.Window` parameter because it's never used and it's not the image importer's responsibility to show dialogs.